src/Microsoft.CodeAnalysis.NetAnalyzers/src/Microsoft.CodeAnalysis.NetAnalyzers/Microsoft.CodeQuality.Analyzers/Maintainability/UseCrossPlatformIntrinsicsAnalyzer.cs

```diff
@@ -140,169 +204,357 @@ private void OnCompilationStart(CompilationStartAnalysisContext context)
+                // Note: MultiplyAdd(x, y, z) expands to (y * z) + x - needs operator expansion
+                // TODO: Implement MultiplyAdd → operator expansion transformation
+                // Note: OrNot expands to two operators - needs operator expansion  
+                // TODO: Implement OrNot → operator expansion transformation
+                AddUnaryMethods(methodSymbols, "RoundToNearest", armAdvSimdTypeSymbolForMethods, RuleKind.Round);
+                AddUnaryMethods(methodSymbols, "RoundToNegativeInfinity", armAdvSimdTypeSymbolForMethods, RuleKind.Floor);
+                AddUnaryMethods(methodSymbols, "RoundToPositiveInfinity", armAdvSimdTypeSymbolForMethods, RuleKind.Ceiling);
+                AddUnaryMethods(methodSymbols, "RoundToZero", armAdvSimdTypeSymbolForMethods, RuleKind.Truncate);
+                // Note: Sqrt is already registered above as operator method
+                AddLoadMethods(methodSymbols, "LoadVector64", armAdvSimdTypeSymbolForMethods, RuleKind.Load);
+                AddLoadMethods(methodSymbols, "LoadVector128", armAdvSimdTypeSymbolForMethods, RuleKind.Load);
+                AddStoreMethods(methodSymbols, "Store", armAdvSimdTypeSymbolForMethods, RuleKind.Store);
+                // VectorTableLookup is 1-to-1 with Shuffle for byte/sbyte
+                AddBinaryMethods(methodSymbols, "VectorTableLookup", armAdvSimdTypeSymbolForMethods, RuleKind.Shuffle, [SpecialType.System_Byte, SpecialType.System_SByte]);
+                // Note: Shuffle/Create APIs like Duplicate, ExtractVector128 need complex transformations (constant extraction)
+            }
+
+            if (compilation.TryGetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemRuntimeIntrinsicsArmAdvSimdArm64, out var armAdvSimdArm64TypeSymbolForMethods))
+            {
+                // Note: AdvSimd.Arm64.Abs for integer types returns unsigned types (e.g., Vector128<long> → Vector128<ulong>),
+                // so we only register AbsScalar for compatible types
+                AddUnaryMethods(methodSymbols, "AbsScalar", armAdvSimdArm64TypeSymbolForMethods, RuleKind.Abs, [SpecialType.System_Double]);
+                AddBinaryMethods(methodSymbols, "AddSaturateScalar", armAdvSimdArm64TypeSymbolForMethods, RuleKind.AddSaturate);
+                // Note: Negate is already registered as op_UnaryNegation above, so we don't register it here
+                // TODO: Add scalar variants of Extract/Insert, FusedMultiply*, Round*
+            }
+
+            if (compilation.TryGetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemRuntimeIntrinsicsWasmPackedSimd, out var wasmPackedSimdTypeSymbolForMethods))
+            {
+                AddUnaryMethods(methodSymbols, "Abs", wasmPackedSimdTypeSymbolForMethods, RuleKind.Abs);
+                AddUnaryMethods(methodSymbols, "Ceiling", wasmPackedSimdTypeSymbolForMethods, RuleKind.Ceiling);
```

Comment:
This is missing at least the following APIs from `PackedSimd`: `AddSaturate`, `AndNot`, `BitwiseSelect`, `Load`, `RoundToNearest`, `Store`, and `SubtractSaturate`